src/sync.js

```diff
@@ -130,12 +130,12 @@ const Sync = async ({ ipfs, log, events, onSynced, start, timeout }) => {
   const peers = new Set()
 
   /**
-   * Event emitter that emits updates.
-   * @name events
+   * Event emitter that emits Sync changes.
    * @†ype EventEmitter
-   * @fires join when a peer has connected and heads were exchanged
-   * @fires leave when a peer disconnects
-   * @fires error when an error occurs
+   * @fires module:Sync~Sync#join when a peer has connected and heads are
+   * exchanged
```

Comment:
This should be "...when a peer has connected and heads were exchanged" (are -> were, because the heads were already exchanged at this point).

Reply:
Suggesting 

"when a peer has connected and the exchange of heads has been completed." 

because it explicitly describes the completion of heads being exchanged.